src/pfe/file-watcher/server/test/scripts/setup.sh

```diff
@@ -6,6 +6,13 @@ RED='\033[0;31m'
 BLUE='\033[0;36m'
 RESET='\033[0m'
 
+CHE_USER="admin"
+CHE_PASS="admin"
+
+KEYCLOAK_HOSTNAME=keycloak-"$NAMESPACE"."$CLUSTER_IP".nip.io
+TOKEN_ENDPOINT="http://${KEYCLOAK_HOSTNAME}/auth/realms/che/protocol/openid-connect/token"
+CHE_ACCESS_TOKEN=$(curl -sSL --data "grant_type=password&client_id=che-public&username=${CHE_USER}&password=${CHE_PASS}" ${TOKEN_ENDPOINT} | jq -r '.access_token')
```

Comment:
I think we do it like this when generating the first time as well. I am wondering if we can make it a global function on the init script and call it every time we want regenerate one?

Reply:
will do

Reply:
@ssh24 I've updated the PR, can you PTAL